### ISSUE TITLE
C# library for aiconfig

### ISSUE THREAD

**Issue by ranjancse26** [lastmile-ai/aiconfig#431]
Request - Please incorporate in building the C# library for AiConfig. I am sure that it will certainly help the large audiences who primarily work with the Microsoft Technologies.

**Comment by saqadri** [comment]
@ranjancse26 thanks for the feedback -- we are going to look at adding an AIConfig server (REST endpoints that wrap around the Python implementation). Then a C# library can be very lightweight wrapper just making REST calls to the python server. Would that suffice? That's the fastest way to add other language support.

**Comment by ranjancse26** [comment]
Yes, agree with you. Restful API is what is needed. Thanks for support

**Comment by saqadri** [comment]
@ranjancse26 we will try to get to it in the next few weeks. Will update this issue when we start work. Thanks for your feedback and please keep the feedback coming
